backend/src/databasefunctions.ts

```diff
@@ -0,0 +1,71 @@
+import * as pg from "pg"
+require("dotenv").config()
+
+
+
+const pgconfig: pg.PoolConfig = {
+  user: process.env.DBUSER,
+  database: process.env.DATABASENAME,
+  password: process.env.DBPASSWORD,
+  host: process.env.DBHOST,
+  port: parseInt(process.env.DBPORT)
+}
+
+const pool = new pg.Pool(pgconfig)
+
+
+pool.on("error", (err: Error,client: pg.PoolClient) => {
+    console.error(err,client)
+})
+
+
+export const createTable = async (sqlStatement: string) => {
+  if (!sqlStatement.toLowerCase().includes("create table")) {
+    throw new Error("wrong type of SQL statement")
+  }
+  const client = await pool.connect()
```

Comment:
This line might also throw an error. it should also be handled in a try-catch block
Same goes for line 28, 43 and 60.

An example is if line 43 throws when inserting a new access token. The frontend will still receive a 200 OK response.